src/Contracts/Parallel.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenAI\Contracts;
+
+use OpenAI\Exceptions\ErrorException;
+use OpenAI\Exceptions\TransporterException;
+use OpenAI\Exceptions\UnserializableResponse;
+
+/**
+ * @template TResponse of Response[]
+ *
+ * @interal
+ */
+interface Parallel
+{
+    /**
+     * Queue up parallel requests.
+     *
+     * @param  array<string, mixed>  $parameters  The OpenAI request parameters.
+     * @param  array-key|null  $key  The index to store the promise in.
+     * @return $this
+     */
+    public function createParallel(array $parameters, int|string|null $key = null): self;
+
+    /**
+     * Execute all parallel requests.
+     *
+     * @return TResponse
```

Comment:
Usually, the @return should be the last line in the docblock..

Reply:
[Just tried updating](https://github.com/defunctl/openai-php-client/pull/1/commits/1ca473e391273f2b484a7b05efe8395488543a33#diff-a1ce3175999541f77405435ed4489053bdc8e66f9c766e4d9ef7bf1e6476842cR34) this and `pint` fails and auto fixes this.

![image](https://user-images.githubusercontent.com/1066195/207660572-7fa81956-997b-4501-b350-3399b1608095.png)